Library/Formula/whois.rb

```diff
@@ -0,0 +1,45 @@
+class Whois < Formula
+  desc "Lookup tool for domain names and other internet resources"
+  homepage "https://github.com/rfc1036/whois"
+  url "https://ftp.debian.org/debian/pool/main/w/whois/whois_5.5.20.tar.xz"
+  sha256 "42085102dfad82067abe2d5d1cfca59586573dee528718559b022e762bb85cf1"
+  license "GPL-2.0-or-later"
+
+  bottle do
+  end
+
+  keg_only :provided_by_osx
+
+  depends_on "make" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "bash-completion"
```

Comment:
Based on your description, is this just a buildtime dependency?

Reply:
No, it actually installs bash completion files. If it can't enumerate the path via pkg-config it defaults to `/etc/bash-completion` & fails due to permissions.